src/GNDStk/basic/src/child.hpp

```diff
@@ -116,7 +116,7 @@ inline const child_t<void,allow::many>
    axis                            ("axis"                            );
 inline const child_t<void,allow::one>
    background                      ("background"                      );
-inline const child_t<void,allow::one>
+inline const child_t<void,allow::one> // <== fixme should this be a many?
```

Comment:
Good question, we'll have to check in the specs

Reply:
There are probably a handful of these things that are wrong.